### PR TITLE
📝 Detail how to override the default upload media types

### PR DESCRIPTION
**Pull request by Jord-Gui**
### General Contributing:
https://github.com/tinacms/tina-cloud/issues/2078
* [x] Have you followed the guidelines in our [Contributing document](https://github.com/tinacms/tinacms.org/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### All New Content Submissions: (To be confirmed by reviewer)

* [x] Title is short & specific
* [x] Headers are logically ordered & consistent
* [x] Purpose of document is explained in the first paragraph
* [x] Procedures are tested and work
* [x] Any technical concepts are explained or linked to
* [x] Document follows structure from templates
* [x] All links work
* [x] The spelling and grammar checker has been run
* [x] Graphics and images are clear and useful
* [x] Any prerequisites and next steps are defined.

image

**Figure: The default accepted file types**

image

**Figure: Added property**

### CODE REVIEW

File: content/docs/reference/media/repo-based.md [hunk 1]
```diff
@@ -46,14 +46,62 @@ E.g, in our [tina-cloud-starter](https://github.com/tinacms/tina-cloud-starter/t
...
+| Polygon File Format                                        | `model/ply`                                                  | .ply                           |
+| PostScript                                                 | `application/postscript`                                     | .ps, .eps, etc.                |
+| Text Files                                                 | `text/*`                                                     | .txt, .md, etc.                |
+| U3D Mesh                                                   | `model/u3d+mesh`                                             | .u3d                           |
+| USDZ Zip                                                   | `model/vnd.usdz+zip`                                         | .usdz                          |
+| Videos                                                     | `video/*`                                                    | .mp4, .avi, .mkv, etc.         |
+
+
+If you would like to specify your own allowed file types, add the following below the  **`tina`** property
```
Comment (kldavis4) [inline content/docs/reference/media/repo-based.md:80]:
```suggestion
If you would like to specify your own allowed file types, add the following to the  **`media`** property
```
Doesn't necessarily need to be below (or above) the tina property

**Comment by JackDevAU** [comment]
Are there any types that a user could upload that aren't supported on Tina Cloud? 
Should we add a point that not all types are allowed as Tina Cloud only allows certain files?

**Comment by joshbermanssw** [comment]
> Are there any types that a user could upload that aren't supported on Tina Cloud? Should we add a point that not all types are allowed as Tina Cloud only allows certain files?

@JackDevAU good idea, similar issue we ran into with tina.io and some file types disallowed by Tina Cloud

**Comment by Jord-Gui** [comment]
> Are there any types that a user could upload that aren't supported on Tina Cloud? Should we add a point that not all types are allowed as Tina Cloud only allows certain files?

Cc: @joshbermanssw @JackDevAU 

As per https://github.com/tinacms/tinacms/issues/4677#issuecomment-2246952180, what we though was a Tina Cloud issue was just Tina.io using Cloudinary. 

Repo based media should accept everything Cloudinary does as it is stored in GitHub.

**Comment by JackDevAU** [comment]
Hey @Jord-Gui 
Should we add in a column for what the default fields are? or is everything in the table part of the defaults and aren't needed to be added into the `accept` field?

**Comment by Jord-Gui** [comment]
> Hey @Jord-Gui Should we add in a column for what the default fields are? or is everything in the table part of the defaults and aren't needed to be added into the `accept` field?

Yeah so everything in that table is supported by default and if you want to override that, then you use the accept property.

File: content/docs/reference/media/repo-based.mdx [hunk 2]
```diff
@@ -92,8 +140,7 @@ Repo-based media is designed to be used around a single-branch workflow. If your
 * If you only have a single branch with media enabled, the media store will source/upload images to/from that branch.
 * If you have multiple branches with media enabled, then all media will be sourced/uploaded to/from the repository's default branch.
 
-If you are configuring Tina on a non-default branch (and the Tina config has not yet been merged to your default branch), you may see this message in the Tina Cloud media settings:\
-
+If you are configuring Tina on a non-default branch (and the Tina config has not yet been merged to your default branch), you may see this message in the Tina Cloud media settings:\\
```
Comment (JackDevAU) [inline content/docs/reference/media/repo-based.mdx:143]:
```suggestion
If you are configuring Tina on a non-default branch (and the Tina config has not yet been merged to your default branch), you may see this message in the Tina Cloud media settings:
```

File: content/docs/reference/media/repo-based.mdx [hunk 3]
```diff
@@ -92,8 +140,7 @@ Repo-based media is designed to be used around a single-branch workflow. If your
...
 
 ![](https://res.cloudinary.com/forestry-demo/image/upload/v1688478218/tina-io/docs/media-not-configured_lnr5lw.png)\
 \
```
Comment (JackDevAU) [inline content/docs/reference/media/repo-based.mdx:146]:
```suggestion
![](https://res.cloudinary.com/forestry-demo/image/upload/v1688478218/tina-io/docs/media-not-configured_lnr5lw.png)

```

**Review by JackDevAU — APPROVED** [review]

**Pull request merged**